src/core/scene/a-scene.js

```diff
@@ -56,7 +56,13 @@ module.exports = registerElement('a-scene', {
 
     init: {
       value: function () {
-        this.behaviors = [];
+        this.behaviors = {
+          tick: [],
+          tock: [],
+          tickPostRender: [],
+          tockPostRender: []
+        };
+        this.postNeedsNotify = false;
```

Comment:
Yeah, let's just emit right when we add/remove rather than keeping this state var and emitting during render loop